### DISCUSSION TITLE
Windows Defender 😡

### DISCUSSION THREAD

**Discussion by howtodo1** [BishopFox/sliver#1032] [General]
Windows Defender is all knowing, and it is everywhere 😤. It grows everyday, until it accumulates immense power 😔. Please 🙏, destroy it before it becomes too powerful.

**Comment by rkervella** [comment]
Is there a question here?

**Comment by paulkwalton** [comment]
Loader = No Defender problems any more.

**Reply by moloch--** [reply]
This is the way.

**Reply by Reyniel** [reply]
@paulkwalton loader? You mean sideloader?

**Comment by U4I5** [comment]
I think  you just need to learn  obfuscation
